health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/DSSCard.js

```
@@ -0,0 +1,31 @@
+
+const DSSCard = () => {
+  const { t } = useTranslation();
+
+  const generateLink = (labelKey, pathSuffix) => {
+    return {
+      label: t(labelKey),
+      link: `/${window?.contextPath}/employee/dss/${pathSuffix}`,
+    };
+  };
+  
+  let links = [
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'links' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)